### ISSUE TITLE
Multiple actions

### ISSUE THREAD

**Issue by Mykolaichenko** [tomasbjerre/pull-request-notifier-for-bitbucket#121]
Hi!
Can i invoke multiple URL's for one pull request using your plugin? 
Tnx.

**Comment by tomasbjerre** [comment]
Yes. Just add as many notifications as you want, https://github.com/tomasbjerre/pull-request-notifier-for-bitbucket/blob/master/sandbox/all.png

**Comment by Mykolaichenko** [comment]
@tomasbjerre can you explain more deeper what i may do if i need 2 steps:
when developer make pull request:
first step) call to first api to make change
second step)  call to second api to make change

**Comment by Mykolaichenko** [comment]
It must be in explained order, not on the contrary

**Comment by tomasbjerre** [comment]
* Add the first URL as *injection url*
* Add the second URL  in the URL field.

If you need some output of the first URL as input of the second you can use the ${INJECTION_URL_VALUE} variable in the second URL.

**Comment by Mykolaichenko** [comment]
@tomasbjerre oh, as i know to injection url i can add url with GET request. I'm working with API and need to use PUT.  But it's very intelligent solution :)

**Comment by Mykolaichenko** [comment]
maybe i need to use wrapper, such rundeck or something other for triggering and use some logic there?

**Comment by tomasbjerre** [comment]
Yes. I can see your use case but i will probably not have time to implement
that any time soon.
Den 2 jun 2016 17:38 skrev "Mykolaichenko" <notifications@github.com>:
